manifests/install.pp

```diff
@@ -1,5 +1,61 @@
-class flapjack::install {
+# = Class: flapjack::install
+#
+# Install FlapJack
+#
+class flapjack::install(
+  $version = 'latest'
```

Comment:
Sorry, thought I replied to this one earlier.

> It may eventuate that we decide to publish 2.0 to different package repositories, in which case latest may be fine. 

We will be publishing 2.0 to different package repositories.